src/middlewared/middlewared/plugins/system/product.py

```diff
@@ -206,6 +206,12 @@ def license_update(self, license_):
             self.middleware.call_hook('system.post_license_update', prev_product_type=prev_product_type), wait=False,
         )
 
+    @private
+    def license_string(self):
```

Comment:
There is already a @staticmethod in this class (`_get_license`) that could be extended to provide this functionality.

I'd rather a new keyword that will return the string and the deserialized license as a tuple from the private `system.license` endpoint instead of adding a new method and calling both of them in a weird way like you're doing.

i.e. something like (license_string, license_dict = system.license True)

```
def license(self, return_license_string_too=False):
    ....
    if return_license_string_too:
        return licenseobj, license_
    return license_